### DISCUSSION TITLE
Auto restore effecting fullscreen windows and alt+tabbing

### DISCUSSION THREAD

**Discussion by rativasavitar** [kangyu-california/PersistentWindows#115] [General]
On windows 7 anytime I have auto restore enabled, fullscreen programs do not minimize when alt+tabbed, and attempting to do so also breaks alt+tabbing and it no longer works unless I pause auto restore and relaunch explorer.exe

**Comment by kangyu-california** [comment]
PW uses same set of APIs on both Win10 and Win7,
It is very hard to trace behavior difference caused by OS version upgrade.

One thing you can try is to rebuild PW from source code on Win7,
if that still does not help, the only suggestion is to disable auto restore, and use pause/resume menu or snapshot feature to manually restore.
